sport/app/football/views/wallchart/groupTableEmbed.scala.html

```diff
@@ -0,0 +1,75 @@
+@import implicits.Football._
+@import pa._
+@import views.support.`package`.Seq2zipWithRowInfo
+
+@(page: model.Page, competition: model.Competition, groupStage: football.model.Groups, groupTable: (Round, Seq[LeagueTableEntry]))(implicit request: RequestHeader, context: model.ApplicationContext)
+
+@footballGroup(round: Round, leagueTableEntries: Seq[LeagueTableEntry]) = {
+@defining(groupStage.matchesList(competition, round)) { matches =>
+    <li class="football-group football-group-embed">
```

Comment:
I think for this, you can change the `li` to a `div` and return the html from this point on. We will probably need to copy some of the CSS over to the interactive atom though for apps.